### ISSUE TITLE
Eq. (24.14) (diffusion denoising models), p.822 (printed), edition 2022-05-18

### ISSUE THREAD

**Issue by djmarti** [probml/pml2-book#87]
This is a follow-up of [issue 74](https://github.com/probml/pml2-book/issues/74). Equation (24.14) (after the correction raised by issue 74, about the missing log) is not equal to Eq. (24.13). In Eq. (24.13) the log is on the expectation integral defined in Eq. (24.12). To swap the order of the log and the integral, we need to apply Jensen's inequality, so the correct version for Eq.(24.14) should be the inequality $L \geq  \text{the current r.h.s. of Eq. 24.14}$. The right hand side is already Ł (the ELBO) defined in Eq.(24.15).

In other words, the '=' sign in Eq.(24.14) should be replaced with a $\geq$ sign.

See also Eqs. (11) and (12) in the [*Deep Unsupervised Learning using Nonequilibrium Thermodynamics* paper](https://arxiv.org/abs/1503.03585).

Let me take this opportunity to thank you for writing these two extraordinary volumes.

**Comment by murphyk** [comment]
fixed, thanks. (BTW I am planning to rewrite the diffusion chapter - given the recent explosion of interest in this model family - to make it clearer and more detailed....)

**Comment by djmarti** [comment]
Thanks! Nice to hear that you're going to revise and expand the chapter
(not that it's not good already!).

El dt., 31 de maig 2022, 8:27, Kevin P Murphy ***@***.***> va
escriure:
